_posts/2020-12-19-generate-mini-site-with-metanorma-cli.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "Generate mini site with Metanorma CLI"
+categories: about
+authors:
+  -
+    name: Abu Nashir
+    email: abunashir@gmail.com
+    social_links:
+      - https://github.com/abunashir
+
+excerpt: >-
+    Generate a collection mini site using Metanorma CLI
+---
+
+Each metanorma collection can be published as a mini site, there are couple of
```

Comment:
Don't refer to collections. A metanorma collection is already defined as a Metanorma document composed of other documents, something we are already doing with BIPM, and this functionality is documented as such in the metanorma-cli README. (We haven't quite gotten around to migrating that documentation to metanorma.com.)

What you're describing is a number of documents that are independent of each other, but can still be hosted on a single web directory. Call that a site, but refer to set of documents, not collection. (That set of documents can of course include a collection document!)